### PR TITLE
v1.6.5

### PR DESCRIPTION
**Pull request by angelapwen**

### CODE REVIEW

File: extensions/ql-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## [UNRELEASED]
```
Comment (shati-patel) [inline extensions/ql-vscode/CHANGELOG.md:4]:
```suggestion
```

You can remove this header!

**Review by shati-patel — COMMENTED** [review]
Small tidy-up, otherwise ✔️

File: extensions/ql-vscode/CHANGELOG.md (hunk 1 above)
Comment (angelapwen) [reply extensions/ql-vscode/CHANGELOG.md:4]: Ah ok! I had thought we left it in for future unreleased changes to be added (seems like sometimes it was left in in the past and sometimes not: https://github.com/github/vscode-codeql/pull/1277)

Comment (angelapwen) [reply extensions/ql-vscode/CHANGELOG.md]: Ah cool 😄

**Review by aeisenberg — APPROVED** [review]

**Pull request merged**